.github/workflows/test-code-samples.yaml

```diff
@@ -0,0 +1,31 @@
+name: Test API Docs Code Samples
+
+on:
+  push:
+    paths:
+      - 'apify-api/openapi/code_samples/**'
+  pull_request:
+    paths:
+      - 'apify-api/openapi/code_samples/**'
+
+jobs:
+  run-code-sample-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Get current branch name
+        id: get-branch-name
+        run: echo "branch_name=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_ENV
+
+      - name: Call Code Sample Tester Actor
+        env:
+          APIFY_TOKEN: ${{ secrets.CODE_SAMPLE_TESTER_ACTOR_TOKEN }}
+          BRANCH_NAME: ${{ env.branch_name }}
+        run: |
+          curl --fail-with-body -X POST "https://api.apify.com/v2/acts/X8D0L4wfpO8fCL1uL/run-sync" \
+          -H "Content-Type: application/json" \
+          -H "Authorization: Bearer $APIFY_TOKEN" \
+          -d '{ "branch": "'"$BRANCH_NAME"'" }'
```

Comment:
Will this work on PRs from forks?

Reply:
This PR is from a fork, so I think so

![CleanShot 2024-12-20 at 11 38 15@2x](https://github.com/user-attachments/assets/5fd36a81-8d08-42ea-aec6-d70c3d5f7d22)

Reply:
The run in the console suggests otherwise 
![image](https://github.com/user-attachments/assets/97f0671b-e2e9-4961-91ea-50ca10e48342)

Reply:
Great catch! I've modified the action and the actor so that the owner of the triggering repo is passed

Reply:
Here's the last run with the test: https://console.apify.com/actors/X8D0L4wfpO8fCL1uL/runs/g4B6nhuQZVKlOc9dx#log

Reply:
Also, sorry I deleted that comment because I realised it was wrong, but hadn't seen your second one yet :)